include/dts-functions.sh

```diff
@@ -1815,5 +1727,35 @@ check_if_intel() {
   if [ $cpu_vendor == "GenuineIntel" ]; then
     return 0
   fi
+}
+
+parse_for_arg_return_next(){
+# This function parses a list of arguments (given as a second argument), looks
+# for a specified argument (given as a first argument). In case the specified
+# argument has been found in the list - this function returns (to stdout) the
+# argument, which is on the list after specified one, and a return value 0,
+# otherwise nothing is being printed to stdout and the return value is 1.
+# Arguments:
+# 1. The argument you are searching for like -r for flashrom;
+# 2. Space-separated list of arguments to search in.
+  local _arg="$1"
+  shift
+
+  # This case should be without quotes, to not match literally but as a
```

Comment:
Why do you want to glob match? Can you give examples?

Reply:
Amended [here](https://github.com/Dasharo/dts-scripts/compare/8e94cbc7f99ffe845b06f6f742c9b9863dec795e..992062280bbd17d42f45e14d98c3e682971d4ee4)

Reply:
Thanks. I wasn't sure how it was used and usually it's probably better to glob explicitly e.g.
`*"$_arg")` instead of via passed variable. Much easier to deduce what's going on.